vilebot/src/main/java/com/oldterns/vilebot/handlers/user/RemindMe.java

```diff
@@ -0,0 +1,54 @@
+package com.oldterns.vilebot.handlers.user;
+
+import ca.szc.keratin.bot.annotation.HandlerContainer;
+import ca.szc.keratin.core.event.message.recieve.ReceivePrivmsg;
+import net.engio.mbassy.listener.Handler;
+
+import java.util.Calendar;
+import java.util.Timer;
+import java.util.TimerTask;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+@HandlerContainer
+public class RemindMe
+{
+
+    private static final Pattern remindMePattern = Pattern.compile( "^!remindme (.+) (.+)" );
```

Comment:
To avoid errors in parsing the number of minutes you could use `(\\d+)` instead of `(.+)`